cmd/startApi.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"github.com/cloud-native-toolkit/itzcli/api"
+	"github.com/spf13/cobra"
+)
+
+// startCmd represents the start command
+var startCmd = &cobra.Command{
+	Use:   "start",
+	Short: "A brief description of your command",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		api.StartServer(RootCmd)
+	},
+}
+
+func init() {
+	apiCmd.AddCommand(startCmd)
+
+	// Here you will define your flags and configuration settings.
+
```

Comment:
I try (but don't always succeed) to delete all the commented-out code before committing to not confuse people.